Backend/app/controllers/customer.controller.js

```diff
@@ -14,6 +14,21 @@ const getAllUsers = ((req, res) => {
     });
 });
 
+const getCases = ((req, res) => {
+    const clientId = req.query.clientId;
```

Comment:
Use `user_id` instead of `clientId` for consistency throughout the API

Reply:
Good point! Thanks Youssef